src/core/aux_structs.jl

```diff
@@ -3,4 +3,8 @@ struct UpdateRef{T}
     access_ref::Union{Symbol,String}
 end
 
+function UpdateRef{T}(name::AbstractString, ::Type{U}) where {T, U <: PSY.Device}
```

Comment:
I'm thinking that we should one or more explicit constructors for UpdateRef.  Users will need better advice on how to create UpdateRef instances.
1. This current one is used for storing ParameterJuMP variables in the format (<basename>_<device_type>).
2. The code creates UpdateRef instances with a string where that string is an accessor function for a forecast. We could create a specific constructor for this purpose.
3. Symbols are passed in other cases that I'm still going through. I'm not yet sure if we need those.

Reply:
I agree with the idea to have functions that return the UpdateRef objects. These were never meant to be used directly by users. Since we have discussed the use of get and set functions I never intended for a user to have to write an UpdateRef object manually.